README.md

```diff
@@ -179,6 +179,8 @@ https://github.com/mattlewis92/angular-bootstrap-calendar
 
 Run `npm start` to start a development server on port 8000 with auto reload + tests.
 
+> To run this on a windows machine change `npm start` in `package.json` from `node --max_old_space_size=8192 ./node_modules/.bin/ng serve --open` to `node --max_old_space_size=8192 ./node_modules/@angular/cli/bin/ng serve --open`.
```

Comment:
Sorry for the friction here! I just pushed [a commit](https://github.com/mattlewis92/angular-calendar/commit/f5b50e87bc0e980887c268cc015abdd6a9666856) that should get rid of this issue, can you let me know if it works ok for you?

Reply:
True. I added it as it looked like the convention but indeed DayJs uses the same API as Moment for those functions. 
Maybe if this changes in the future we can readd it.

Reply:
Worked like a charm! Removed the lines from the doc.